<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="com.Automation.Test.Automation_Test" duration-ms="3097" started-at="2019-04-11T17:33:39Z" finished-at="2019-04-11T17:33:42Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="3097" started-at="2019-04-11T17:33:39Z" finished-at="2019-04-11T17:33:42Z">
      <class name="com.Automation.Test.Automation_Test">
        <test-method status="PASS" signature="beforeAllClassesStare()[pri:0, instance:com.Automation.Test.Automation_Test@b97c004]" name="beforeAllClassesStare" is-config="true" duration-ms="12" started-at="2019-04-11T13:33:39Z" finished-at="2019-04-11T13:33:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeAllClassesStare -->
        <test-method status="PASS" signature="beforeEachTestStart()[pri:0, instance:com.Automation.Test.Automation_Test@b97c004]" name="beforeEachTestStart" is-config="true" duration-ms="9" started-at="2019-04-11T13:33:39Z" finished-at="2019-04-11T13:33:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeEachTestStart -->
        <test-method status="FAIL" signature="method_call()[pri:0, instance:com.Automation.Test.Automation_Test@b97c004]" name="method_call" duration-ms="29" started-at="2019-04-11T13:33:39Z" finished-at="2019-04-11T13:33:39Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.Automatiom.Library.seleniumRepository.scrollByOffsetVertical(seleniumRepository.java:226)
	at com.Automation.Pages.C_CheckOut_page.checkout(C_CheckOut_page.java:13)
	at com.Automation.Test.Automation_Test.method_call(Automation_Test.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method_call -->
        <test-method status="PASS" signature="afterEachTestEnd()[pri:0, instance:com.Automation.Test.Automation_Test@b97c004]" name="afterEachTestEnd" is-config="true" duration-ms="3026" started-at="2019-04-11T13:33:39Z" finished-at="2019-04-11T13:33:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterEachTestEnd -->
        <test-method status="PASS" signature="afterAllTestCompleted()[pri:0, instance:com.Automation.Test.Automation_Test@b97c004]" name="afterAllTestCompleted" is-config="true" duration-ms="1" started-at="2019-04-11T13:33:42Z" finished-at="2019-04-11T13:33:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterAllTestCompleted -->
      </class> <!-- com.Automation.Test.Automation_Test -->
    </test> <!-- Command line test -->
  </suite> <!-- com.Automation.Test.Automation_Test -->
</testng-results>
